open-company/open-company-web
# Growth editing

card: https://trello.com/c/4dZnM5rp

test:
- try to add the growth section for a company that doesn't have it yet
- try to add a new metric to a company that already have the growth section
- try to edit the data and the metadata for an existing metric
- check CI
- merge

NB: there is a problem with cljs-time https://github.com/andrewmcveigh/cljs-time/issues/55
